azureadps-2.0/AzureAD/Get-AzureADMSApplicationOwner.yml

```diff
@@ -28,7 +28,7 @@ examples:
   summary: ""
 - title: 'Example 1: Get the owners of an application'
   code: |-
-    PS C:\>Get-AzureADMSApplicationOwner -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -All $true
+    PS C:\>Get-AzureADMSApplicationOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -All $true
```

Comment:
GUIDs in the output haven't been updated.

Reply:
Changes implemented here are ok for the purposes of this PR